--- NagiosGrapher-1.6.1-rc5-0.5/Makefile.in	2008-12-01 16:35:33.707580782 +0200
+++ NagiosGrapher-1.6.1-rc5-0.5-hash/Makefile.in	2008-12-01 16:33:40.667603355 +0200
@@ -142,7 +142,7 @@
 	@echo "pre 1.6 Users Attention!"
 	@echo "-----------------------"
 	@echo "The icon_image_tag changed from version! Please change it to:"
-	@echo "   icon_image_tag  ' dot.png' border=\"0\"></a><A TARGET=\"_blank\" HREF=\"graphs.cgi?###URL###\"><img src='###IMAGESRC###' '"
+	@echo "   icon_image_tag  ' dot.png' border=\"0\"></a><A TARGET=\"_blank\" HREF=\"graphs.cgi?___URL___\"><img src='___IMAGESRC___'"
 	@echo
 
 fixdeps-messages:
--- NagiosGrapher-1.6.1-rc5-0.5/bin/collect2.pl.in	2007-08-10 18:05:31.000000000 +0300
+++ NagiosGrapher-1.6.1-rc5-0.5-hash/bin/collect2.pl.in	2008-12-01 16:34:54.705196959 +0200
@@ -613,18 +613,18 @@
 	                $graph_url     = "host=$host&service=$service";
 
 	                # Replace wrappers misc
-	                $new_image_tag =~ s/###HOST###/$host/gi;
-	                $new_image_tag =~ s/###SERVICE###/$service/gi;
-	                $new_image_tag =~ s/###URL###/$graph_url/gi;
+	                $new_image_tag =~ s/___HOST___/$host/gi;
+	                $new_image_tag =~ s/___SERVICE___/$service/gi;
+	                $new_image_tag =~ s/___URL___/$graph_url/gi;
 
 					# Replace wrappers, src specific
 					if ($image_static) {
-						$new_image_tag =~ s/###IMAGESRC###/$image_src/gi;
+						$new_image_tag =~ s/___IMAGESRC___/$image_src/gi;
 		            }
 		            elsif (!$image_static && $image_script) {
 		            	$new_image_script = $image_script;
-		            	$new_image_script =~ s/###URL###/$graph_url/gi;
-		            	$new_image_tag =~ s/###IMAGESRC###/$new_image_script/gi;
+		            	$new_image_script =~ s/___URL___/$graph_url/gi;
+		            	$new_image_tag =~ s/___IMAGESRC___/$new_image_script/gi;
 		            }
 
 					my %args = (
--- NagiosGrapher-1.6.1-rc5-0.5/cfg/ngraph.ncfg.in	2007-08-10 18:05:31.000000000 +0300
+++ NagiosGrapher-1.6.1-rc5-0.5-hash/cfg/ngraph.ncfg.in	2008-12-01 16:33:55.850917365 +0200
@@ -38,9 +38,9 @@
     nagios_config           @NAGIOS_CONFIG@
     cgi_config              @NAGIOS_CONFIG_CGI@
 
-    icon_image_tag          ' dot.png' border="0"></a><A TARGET="_blank" HREF="graphs.cgi?###URL###"><img src='###IMAGESRC###' '
+    icon_image_tag          ' dot.png' border="0"></a><A TARGET="_blank" HREF="graphs.cgi?___URL___"><img src='___IMAGESRC___' '
     icon_image_src          @NG_LOGOS@/graph.png
-    icon_image_script       @NG_CGI@/rrd2-system.cgi?###URL###&start=-5400&title=Actual&width=20&height=20&type=AVERAGE&only-graph=true
+    icon_image_script       @NG_CGI@/rrd2-system.cgi?___URL___&start=-5400&title=Actual&width=20&height=20&type=AVERAGE&only-graph=true
     icon_image_static       true
 
     log_file                @NG_LOGFILE@
--- NagiosGrapher-1.6.1-rc5-0.5/doc/CONFIG	2006-09-25 18:51:10.000000000 +0300
+++ NagiosGrapher-1.6.1-rc5-0.5-hash/doc/CONFIG	2008-12-01 16:34:15.395032399 +0200
@@ -61,7 +61,7 @@
         cgi_config              /etc/nagios/cgi.cfg
         
         # the syntax for the serviceext entries
-        icon_image_tag          dot.gif' border="0"></a><A TARGET="_blank" HREF="graphs.cgi?###URL###" BORDER="0"><img src="/nagios/images/logos/graph.png" border="0"logos/dot.gif
+        icon_image_tag          dot.gif' border="0"></a><A TARGET="_blank" HREF="graphs.cgi?___URL___" BORDER="0"><img src="/nagios/images/logos/graph.png" border="0"logos/dot.gif
         
         # logfile and loglevel
         log_file                /var/log/nagios/ngraph.log
