--- NagiosGrapher-1.6.1-rc5-0.5/lib/NagiosGrapher.pm.in~	2008-12-01 17:30:58.000000000 +0200
+++ NagiosGrapher-1.6.1-rc5-0.5/lib/NagiosGrapher.pm.in	2008-12-01 17:33:41.358902309 +0200
@@ -570,7 +570,7 @@
 	if ( open( CFG, "<" . $ngraphcfg ) ) {
 		while (<CFG>) {
 			if ( m/\s*define\s+config\s*{/ .. /\s*}/ ) {
-				s/#.*$//;
+				s/^\s*#.*$//;
 				chomp();
 				if (m/\s*define\s+config\s*{/) {
 					# do nothing;
