--- NagiosGrapher-1.6.1-rc5-0.5/sbin/graphs.cgi.in~	2007-08-15 13:23:16.000000000 +0300
+++ NagiosGrapher-1.6.1-rc5-0.5/sbin/graphs.cgi.in	2008-11-30 16:02:07.189748976 +0200
@@ -352,7 +352,7 @@
 }
 
 sub action_showcalendar {
-  use Calendar::Simple;
+  use HTML::Calendar::Simple;
 
   my @today = localtime( time() );
 
--- NagiosGrapher-1.6.1-rc5-0.5/sbin/rrd2-system.cgi.in~	2008-12-01 14:55:05.000000000 +0200
+++ NagiosGrapher-1.6.1-rc5-0.5/sbin/rrd2-system.cgi.in	2008-12-01 14:56:27.971503013 +0200
@@ -82,7 +82,7 @@
 my @caller = ($graph_script, join(';', @param_arr));
 
 # get the image
-my $perl = `which perl`;
+my $perl = '/usr/bin/perl';
 chomp($perl);
 my $image_bin;
 open (IMG, "-|") || exec $perl,@caller;
