diff -uNr NagiosGrapher-1.6-rc1.orig/Makefile.in NagiosGrapher-1.6-rc1/Makefile.in
--- NagiosGrapher-1.6-rc1.orig/Makefile.in	2006-06-07 18:22:37.000000000 +0200
+++ NagiosGrapher-1.6-rc1/Makefile.in	2006-06-21 14:18:40.000000000 +0200
@@ -125,14 +125,11 @@
 
 dirs:
 	mkdir -p $(DESTDIR)/$(NG_SRVEXT_DIR)
-	chown -R $(NAGIOS_PROC_USER) $(DESTDIR)/$(NG_SRVEXT_DIR)	
 	mkdir -p $(DESTDIR)/$(NG_RRD)
-	chown -R $(NAGIOS_PROC_USER) $(DESTDIR)/$(NG_RRD)	
-
+	
 logfile:
 	touch $(DESTDIR)/$(NG_LOGFILE)
-	chown $(NAGIOS_PROC_USER) $(DESTDIR)/$(NG_LOGFILE)
-
+	
 config-install:
 	mkdir -p $(DESTDIR)/$(NG_CONFIG)
 	mkdir -p $(DESTDIR)/$(NG_CONFIG_SUB)
@@ -142,16 +139,9 @@
 	cp -r cfg/templates/extra $(DESTDIR)/$(NG_CONFIG_SUB)
 	cp -r cfg/templates/standard $(DESTDIR)/$(NG_CONFIG_SUB)
 
-	chown -R $(NAGIOS_PROC_USER) $(DESTDIR)/$(NG_CONFIG)/*.ncfg
-	chgrp -R $(NAGIOS_PROC_GROUP) $(DESTDIR)/$(NG_CONFIG)/*.ncfg
-
-
 cgi-install:
 	mkdir -p $(DESTDIR)/$(NAGIOS_FOLDER_CGI)
 	cp *.cgi $(DESTDIR)/$(NAGIOS_FOLDER_CGI)
-	chown -R $(NAGIOS_PROC_USER) $(DESTDIR)/$(NAGIOS_FOLDER_CGI)
-	chgrp -R $(APACHE_GROUP) $(DESTDIR)/$(NAGIOS_FOLDER_CGI)
-	chmod 0755 -R $(DESTDIR)/$(NAGIOS_FOLDER_CGI)
 
 bin-install:
 	mkdir -p $(DESTDIR)/$(NAGIOS_CONTRIBUTION)
@@ -160,10 +150,6 @@
 		contrib/udpecho/udpecho.c \
 		$(DESTDIR)/$(NAGIOS_CONTRIBUTION)
 
-	chown -R $(NAGIOS_PROC_USER) $(DESTDIR)/$(NAGIOS_CONTRIBUTION)
-	chgrp -R $(NAGIOS_PROC_GROUP) $(DESTDIR)/$(NAGIOS_CONTRIBUTION)
-	chmod 0755 -R $(DESTDIR)/$(NAGIOS_CONTRIBUTION)
-
 bin-compile:
 	gcc $(DESTDIR)/$(NAGIOS_CONTRIBUTION)/fifo_write.c -o \
 		$(DESTDIR)/$(NAGIOS_CONTRIBUTION)/fifo_write
@@ -171,16 +157,12 @@
 	gcc $(DESTDIR)/$(NAGIOS_CONTRIBUTION)/udpecho.c -o \
 		$(DESTDIR)/$(NAGIOS_CONTRIBUTION)/udpecho
 
-	chown -R $(NAGIOS_PROC_USER) $(DESTDIR)/$(NAGIOS_CONTRIBUTION)
-	chgrp -R $(NAGIOS_PROC_GROUP) $(DESTDIR)/$(NAGIOS_CONTRIBUTION)
-
 perl-install:
 	mkdir -p $(DESTDIR)/$(PERL_INC)
 	cp NagiosGrapher.pm $(DESTDIR)/$(PERL_INC)
 
 init-script:
 	cp nagios_grapher /etc/init.d
-	chmod 0755 /etc/init.d/nagios_grapher 
 
 image-install:
 	mkdir -p $(DESTDIR)/$(NAGIOS_IMAGES)
