diff -uNr NagiosGrapher-1.6-rc1.orig/config.layout NagiosGrapher-1.6-rc1/config.layout
--- NagiosGrapher-1.6-rc1.orig/config.layout	2006-05-29 16:23:07.000000000 +0200
+++ NagiosGrapher-1.6-rc1/config.layout	2006-06-21 18:15:31.000000000 +0200
@@ -20,6 +20,26 @@
   ng_rrd:				${prefix}/var/rrd
 </Layout>
 
+# PLD layout
+<Layout PLD>
+  prefix:				/usr/lib/nagios
+  nagios_config:			/etc/nagios/nagios.cfg
+  nagios_config_cgi:			/etc/nagios/cgi.cfg
+  nagios_images:			${datadir}/nagios/images
+  nagios_images_logos:			${datadir}/nagios/images
+  nagios_folder_cgi:			${libdir}/nagios/cgi 
+  perl_inc:				${datadir}/perl5/vendor_perl
+  nagios_contribution:			${libdir}/nagios/grapher
+  ng_config:				/etc/nagios
+  ng_config_sub:			${ng_config}/ngraph.d
+  ng_daemon:				/var/lib/nagios/nagios_grapher
+  ng_srvext_file:			/etc/nagios/serviceext.cfg
+  ng_srvext_dir:			/etc/nagios/serviceext
+  ng_interface_pipe:			/var/lib/nagios/rw/ngraph.pipe
+  ng_logfile:				/var/log/nagios/ngraph.log
+  ng_rrd:				/var/lib/nagios/rrd
+</Layout>
+
 # layout for debian package install
 <Layout debian>
   prefix:				/usr/local/nagios
